bin/ream/src/cli/account_manager.rs

```diff
@@ -14,23 +17,35 @@ pub struct AccountManagerConfig {
 
     /// Account lifetime in 2 ** lifetime slots
     #[arg(short, long, default_value_t = 18)]
-    pub lifetime: u64,
+    pub lifetime: u32,
 
     /// Chunk size for messages
     #[arg(short, long, default_value_t = 5)]
-    pub chunk_size: u64,
+    pub chunk_size: u32,
 
     /// Seed phrase for key generation
-    #[arg(short, long)]
+    #[arg(short, long, conflicts_with = "import_keystore")]
     pub seed_phrase: Option<String>,
 
+    /// Optional BIP39 passphrase used with the seed phrase
+    #[arg(long)]
+    pub passphrase: Option<String>,
+
     /// Activation epoch for the validator
     #[arg(long, default_value_t = DEFAULT_ACTIVATION_EPOCH)]
-    pub activation_epoch: usize,
+    pub activation_epoch: u32,
 
     /// Number of active epochs
     #[arg(long, default_value_t = DEFAULT_NUM_ACTIVE_EPOCHS)]
-    pub num_active_epochs: usize,
+    pub num_active_epochs: u32,
+
+    /// Path for keystore directory
+    #[arg(long, default_value = DEFAULT_KEYSTORE_PATH)]
+    pub path: String,
+
+    /// Import existing keystore
+    #[arg(long, conflicts_with = "seed_phrase")]
+    pub import_keystore: Option<String>,
```

Comment:
```suggestion
    pub import_keystore: Option<PathBuf>,
```

if this is a supposed to respond to a Path, it shouldn't be a string